packages/mdc-radio/package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@material/animation": "^0.41.0",
     "@material/base": "^0.41.0",
+    "@material/dom": "^0.41.0",
```

Comment:
Remove this dependency now that `EventType` and `SpecificEventListener` are now in the `mdc-base` package.

Reply:
We'll do this in PR #4358.